tests/test_optimizer/unittest_cpu_adam.py

```diff
@@ -121,7 +88,7 @@ def check_res(
             p_grad.view(-1),    # fp32 grad
             exp_avg.view(-1),
             exp_avg_sq.view(-1),
-            loss_scale,
+            -1,
```

Comment:
why we set loss_scale as -1?

Reply:
oh, I just forget to change it...
it should be -1 or 2**5

Reply:
Have fixed it.